src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -11152,14 +11007,14 @@ private ExpressionSyntax ParsePostFixExpression(ExpressionSyntax expr)
                                 expr,
                                 this.EatToken(),
                                 ParseConsequenceSyntax());
-                            break;
+                            continue;
                         }
 
-                        goto default;
+                        return expr;
 
                     case SyntaxKind.ExclamationToken:
                         expr = _syntaxFactory.PostfixUnaryExpression(SyntaxKind.SuppressNullableWarningExpression, expr, this.EatToken());
-                        break;
+                        continue;
```

Comment:
It looks like using continue here instead of break is just to improve clarity? We used to jump to the end of the switch statement and then hit the top of the next loop iteration, but now we just hit the top of the next loop iteration?

Reply:
correct.  IMO 'continue' is better for clarity.  'break' both has the issue of being confusing (am i breaking the loop?  or am i in a switch?), and needing the reader to then need to see what happens at the end of hte loop.  'continue' perfect describes what is happening, and doesn't require forward reading.